src/tagstr_site/htm.py

```diff
@@ -300,14 +303,39 @@ def html(*args: str | Interpolation) -> HTML:
         '</body></html>',
     )
 
-    this_parser = ASTParser()
-    for this_arg in these_args:
-        this_parser.feed(this_arg)
+    parser = AstParser()
+    for arg in args:
```

Comment:
I was doing `this_arg` because PyCharm complains when you have variable names shadowing an outer scope.